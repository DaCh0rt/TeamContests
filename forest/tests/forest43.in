982 69
-361 6148
2219 -7452
6390 4835
-150 -7321
1832 5236
-5824 8144
-187 -4022
3152 -5284
9090 -3216
7932 -2974
9382 3783
-7387 -5807
-9809 8390
-7185 97
2905 1603
-1886 -2575
-411 8970
6635 -8888
-5665 -5241
1455 7167
-4012 -8693
567 8545
7673 1718
896 -2127
4318 3476
7375 -9420
-292 1876
-1944 7726
-7426 -1126
-1533 -680
9877 -5619
787 -3811
3859 995
6094 -5676
-1430 1033
-7151 -6395
5826 -8265
-6061 -6087
-1223 6484
4449 2711
6533 -3929
7591 7447
-5565 -4800
6770 -4262
3436 997
2706 5004
-6135 -3398
-7835 -4525
-414 -4357
-7365 4078
-6639 -5452
4300 7361
5385 -2285
5159 -6470
891 4264
7761 6137
-8753 -7515
-5563 -4129
466 9874
-8999 -6459
2882 -515
-8095 45
-5456 9346
-2015 -1184
-1125 -1924
-8867 1307
-7904 -31
-9946 -5107
-9251 -3145
-5897 -9354
-8265 -4773
5718 -3487
4631 9765
-5071 -6123
-6440 -8817
3362 5309
-4103 -5921
8138 6501
7208 -981
-5987 -4592
431 -2537
2518 5985
-6077 -3364
9173 1735
5485 6605
1130 918
-7870 9649
6392 -4879
-558 -1873
3002 5238
6896 7423
8783 2777
530 -4562
8345 1768
1226 1755
4540 2015
9046 -8592
1544 -8709
-4078 -9414
4529 -1888
-7883 -5085
2489 8178
-1249 1679
89 -4540
-5328 7155
2528 -4762
5950 1021
-7993 5030
8573 -9965
4314 -6056
-4472 2558
81 4920
-1812 5438
1442 739
-7268 5819
-618 9237
-1282 -8404
4263 2201
-4824 9668
-1649 5220
-1038 8226
-2752 8473
2380 8262
-3554 -6234
5886 9183
97 -3745
2104 -4873
9066 -3786
2205 6249
3263 -5190
3719 8575
-7347 1994
4048 8070
-4516 -243
-9584 8251
1945 -166
8610 -2683
-2226 -5732
-6053 2764
-1635 -74
-4005 9878
2411 713
-1605 -5287
3488 5140
8265 2282
7135 4340
3738 853
1363 9222
3741 1927
-3999 7909
-2987 -2964
5533 -873
9334 7065
-1299 3921
-6098 -7092
-5508 9481
7077 -4506
6619 3708
7495 -5708
7970 -2001
-1267 7737
-2360 -6436
1298 -1568
7807 219
3642 -280
-6305 4350
-7625 4561
1365 3745
2141 -8195
-3345 3688
2494 1851
589 -6338
3460 -2631
-8752 4852
-2856 -7216
7088 3486
727 -8938
3129 -9082
617 -637
-4697 -9353
-4673 3027
-1491 -4092
1751 9756
3179 6422
-2497 4805
-7896 757
-8590 -4701
-960 -1177
7129 -3875
-8532 1222
9042 151
-7807 -806
-1718 -656
-6057 -2899
3228 -4885
4783 2378
-5023 -6064
6790 -6935
966 8224
-472 -172
-9157 3271
5737 9167
8624 -6229
9079 6804
-974 -2013
-3868 7615
9136 3259
-5848 -8341
-2615 -3823
7061 68
255 -5526
9687 -9439
-6261 -2658
7171 -1374
6719 4090
-4937 2701
-1076 -7083
-4404 -6158
8270 -4241
-6984 -9200
6803 9994
2821 -3196
8664 3791
978 4157
9973 9913
-196 -4872
-981 -454
-5151 -1699
-6752 -4704
3668 -3319
3988 -6911
4993 8447
7715 -8334
9100 1951
8782 4129
-6680 -4168
-5527 4093
-8607 -8551
7635 -5079
9453 3781
-9535 -9955
6105 -8383
1485 5963
1702 4864
-2705 -5894
6140 5010
7598 2351
6513 -7488
1938 3301
8749 -1362
-791 5137
-3229 386
6924 -4550
4703 1817
-7946 3906
-9782 18
2 -9380
3607 -3655
210 1694
3757 -5991
5134 -3534
-852 44
9603 -7687
477 -1252
-9002 7395
-8146 -130
7761 6633
7454 -7567
-4453 -4241
7575 290
5931 -2383
-5176 4230
1296 5630
5443 -5786
5700 -7110
8722 3494
-5742 6845
-2065 3248
-4018 1019
7106 -7884
-3049 -5716
3445 -9692
-3099 -6869
1956 -594
-5647 -8757
-2763 2429
1202 -9210
2228 -1814
4869 -6136
9859 8151
6751 -1324
6735 -9039
5879 1685
937 8930
6483 -9812
-5855 -6560
-4483 9494
-1920 -7812
-7781 -5215
-2076 1539
8604 4538
9826 -1482
3056 4201
-4347 6383
1545 -3664
-5204 -7788
1395 3688
-330 -3001
9737 -3486
-2716 -9658
-6618 455
4344 -7248
9146 -2162
991 7378
-7669 1335
1972 -7156
-8793 -1399
-4268 -9386
5600 2343
7660 7935
1911 5823
9897 -5220
-8107 9580
8464 4555
4188 9140
-9455 -8137
-4608 -5656
-4611 -2668
2837 -3538
1325 -8657
-8093 4303
-543 5900
7724 -8522
-2374 7744
-4065 -7402
7053 2067
4259 -4827
-6939 2727
3135 -5584
167 8962
-6793 5245
-9763 2665
-5318 6088
-8350 212
9609 -3742
9666 -9868
7852 5133
7342 3032
-7232 6521
5994 5724
-9968 -2119
-7589 -5961
7583 -5178
-3810 9800
9538 4154
8486 -6144
-451 5819
-7706 6617
-1217 5359
-7889 -4310
142 6547
817 7610
-1128 -8217
5315 879
1354 -662
-9980 -8328
9690 2290
1195 -5203
7256 -8723
-6289 1221
-8437 -3956
-3431 -8871
4139 -5754
5650 -2514
4623 4014
56 3255
-1882 -8565
1401 9757
2252 -3669
8018 -6135
7594 -880
-8964 3714
-2884 -7525
8206 -7691
8438 6103
-6784 -6312
-5369 4463
6416 1935
2251 8275
-2575 -3751
-193 3791
-807 -2347
-110 2413
1572 6169
-3726 487
9034 -4400
6137 8395
-4574 -5966
426 -6622
6049 -7857
5983 3491
-9214 -7769
2145 -8309
-912 -2462
9739 -8947
-200 -4869
5903 -7879
2402 58
7922 -6812
5450 -978
9734 3620
-4384 9466
8178 8922
6911 -695
-7546 3334
6066 -6044
-1082 9606
-4500 3768
9470 -1092
6857 9032
9425 -364
-1684 9612
-5184 1579
8998 -2596
-452 -790
-2687 8275
1346 6630
2650 -5215
190 6992
1037 2854
1973 -2867
-9312 -7509
-1007 7304
-2482 412
-1269 3518
2853 265
-3120 1941
6233 -835
5477 -5697
-5200 6172
5703 -325
-7036 -8561
9374 2291
-8303 -12
750 -3622
-364 3394
4789 4971
7981 7522
-3433 7028
-8485 9948
-1308 -3973
-8716 -1314
3627 9476
-8982 7820
8859 -2103
-9115 -3993
-2157 -6531
989 4389
-5632 6351
-1088 -584
5611 -7958
8341 -5541
9473 -4072
-7900 -8243
-6000 2300
-4390 -6846
-5119 -2237
5764 -3442
6663 6473
-9030 -2863
6300 -4036
8743 -7957
-2426 -8709
-7544 7996
5588 -5770
-6199 4267
-4848 -3536
8213 -5304
-670 8731
-9395 -3791
-3877 7061
6568 3248
9530 5914
-9597 383
9455 -2163
7245 -5227
5374 991
942 -5275
803 890
8876 6140
3609 -2469
-1154 3292
-7877 4303
-2931 3854
-5690 5341
5176 8795
-4281 -6462
9544 584
-2467 -2146
-1065 -6152
-2991 9278
9863 6878
-4544 -7266
-217 -9484
-4604 -3959
8342 9873
-4897 7136
4763 6097
6553 -6087
2383 2325
5909 -6573
781 3297
-8492 1822
3306 -9271
-6425 1447
7568 -1570
9718 2829
2789 2614
-3473 4683
-2107 -1754
-9668 3854
3279 -290
1342 -9184
7274 2892
1142 1845
3883 4322
6261 -941
6338 -320
4299 -2222
2654 -4283
-2063 -635
9608 2708
-7243 -7197
9423 -1323
1409 -1839
-9175 -6213
-9721 -2629
1217 -2385
1163 -9187
5252 8925
2807 -7328
9493 -9096
3087 -487
5693 -3118
-5676 1981
-1584 -9324
2824 -9520
3726 -8328
-6125 -2927
3018 5553
-6109 -7491
2097 3799
-107 -5576
-4985 2519
3897 3009
-8464 3698
-6794 8421
-2854 228
401 1954
2235 1132
-574 -2482
-7513 5872
-9380 9334
-9695 444
9049 -4453
-4077 4255
-6667 -279
-2242 1834
9527 -5317
3936 -5427
6634 -1896
8870 2020
3966 9248
-7607 5596
-9457 2941
5138 -4450
-6898 -7975
-976 -7679
9541 -395
-6429 4261
1122 9306
-4509 9067
-3746 2517
8549 7229
-118 -5451
5662 -3603
6880 -1178
2128 8598
7061 9516
9609 4956
-9568 -5805
5505 -7959
-9819 -6224
9847 -4792
3582 4145
5605 -5173
8997 -109
8212 -3355
3333 6480
5565 8973
766 -1787
3370 -7327
648 3173
-3857 -3800
-84 -1652
6181 -4279
504 -6266
-5723 3651
8466 2767
-2148 -9934
399 -2949
8991 -6733
1899 3597
2777 8845
-9101 -2736
8874 -4592
-8125 8477
2744 4804
-6146 6155
-5457 5568
8620 8009
-853 -6724
1891 731
2262 -7827
8686 -3973
5018 5983
-2536 -9919
6759 -7333
-6915 -1504
-8093 7546
4436 -2641
-2385 -8213
-9180 -7606
3247 2905
7971 3379
8281 8984
-2657 6164
1369 2003
-6601 7056
9382 498
-1077 -6534
-7711 1340
-2454 -3484
9586 3489
-5579 -283
7949 -8065
459 6420
7935 -8893
7147 5500
-4183 -8384
4284 -5651
-1377 6091
-9151 -5956
7201 6195
8718 5412
9478 8367
-3296 5517
-3394 1596
7351 9364
-5022 7795
3071 -6616
-1062 2788
-593 -6919
-5315 -5787
42 8018
8869 -292
4847 -7019
5830 8791
-6103 -1641
4765 7250
7641 -525
-6365 -196
-3018 -4795
-6118 9565
-9371 2045
-9728 -3332
4686 7246
-7297 2837
-4337 -6165
-1908 3345
7874 2242
-7807 223
6849 4637
-5367 8590
5130 -7107
7050 -2892
-1593 -1975
-2993 -6154
3343 9662
4206 -8389
-2161 5806
2416 -5991
-6017 6052
-5288 404
5240 9045
-3617 5454
-7146 -1563
-8882 -8882
-1964 -9012
-4492 -1063
527 9999
-3041 7754
3439 -8853
-5390 4712
7402 9004
-9300 5466
-7217 6382
-7911 8974
-5538 1912
4088 -5608
-6930 3534
5201 -8802
4422 -4269
1228 -6485
8380 -7739
-8721 -1582
4595 -1997
-963 -1271
-2123 -9500
1732 -7051
-9512 1778
-4959 -4041
2565 -8999
8657 272
-5337 8910
6862 5028
-7978 -8412
8323 9181
-8616 9777
2362 -419
6323 -7080
-9044 -224
5418 -2411
1378 7042
9183 -2591
4863 3104
-52 3251
6479 -4159
4508 -8734
-7568 4448
6968 -8416
-7713 7493
-7840 5605
-9408 1153
5795 1871
8527 1029
7276 -9132
4582 7205
6832 -3229
-1506 -2895
-3709 5739
3270 110
1746 -6181
6682 8183
-2407 7355
1494 4057
263 912
-4359 7422
8185 -95
3109 -9756
-6197 -350
3727 -1155
-4299 1943
-8351 -3097
-3185 -9622
974 -9890
-9405 6362
8735 -7300
-4672 -6022
8284 6170
1784 -5028
2446 5800
-8003 -1905
1800 4605
8833 -2454
-3018 8438
3839 433
302 2410
7810 -4568
-6527 6947
-6210 7462
-1007 6731
8907 -793
7980 2226
-7488 -2173
8160 7486
7337 -1929
5104 7526
-9429 -220
-3877 -8433
-4584 5818
-8538 8546
-1919 7935
8826 -672
9064 5401
7855 -9099
8453 -3770
-3766 -6060
-6201 4283
-1257 -1764
8647 -3515
-5876 -2704
5526 5967
-1511 7504
9069 -3111
698 -2030
276 1621
6916 -277
-4392 -4171
-9071 8563
-44 -3988
-3987 -5557
2135 -2700
-9280 -3602
-7335 -9559
2142 -238
8394 8211
3655 -6096
-6356 -275
-6229 2754
130 -7080
-7460 -166
3614 -8161
-9100 8710
6626 -9338
-741 1551
-1279 -6791
-517 -416
7459 3544
-407 5959
6383 -243
-5882 3461
5714 -5381
-8297 6997
-9843 -6440
5190 7056
-7485 -9433
9806 8979
-1920 7607
-322 -5264
-2514 -3937
9677 -4982
-1820 -347
-9117 2387
3859 -8529
7318 -4753
6517 1349
6815 1217
4921 -713
1883 7660
-9937 -6405
6681 -8778
5594 -338
-7645 6164
-3381 3721
-8361 -4774
3548 7487
961 9876
-9816 -7174
-4022 357
-7280 9247
-5792 4348
4472 1355
-8328 -1337
-4506 2765
-9825 -8264
-4822 681
4071 -1654
-1018 2515
-935 648
-5093 7711
7306 -4921
-4008 3187
4800 7664
-7632 -1510
4715 4765
5412 -8875
6422 1032
-7282 -3777
1011 -9300
-9717 -9055
4526 4101
6628 5325
2118 -8536
-1070 -1495
4815 -7528
5980 9706
-4465 6986
4915 5309
-2387 -4356
-4409 5806
-1134 -1224
5100 6336
1893 7042
-7493 432
3333 1530
-9550 -1622
-9769 9417
4157 9739
-835 7372
-716 -4485
-3827 -4998
-1503 8726
8440 -6532
-8188 8725
9410 3720
-1169 9049
4498 6321
3334 -185
5523 5368
-3175 -5871
-911 2988
229 -2723
-2580 1727
90 7681
4566 -6444
1108 6074
5368 2057
-1941 -1639
-6985 2313
-7414 5535
-441 -2036
8934 641
914 5686
-8029 2883
-2034 -1241
5213 -5135
9989 -8584
4408 -6563
6391 -5965
-983 2430
7080 -8116
6025 -5273
-9284 7490
9669 -6026
9823 790
-8973 9930
8113 -2400
-9163 805
-4629 2494
7697 3355
249 6134
-9036 8317
-2590 -7863
6980 7921
-423 -4112
-9384 -7156
9974 -7708
7784 -5203
-738 8379
2385 5218
-1134 -1335
-56 -8588
9395 -3729
-1941 -1579
4734 1073
4413 -4851
-6697 -1290
5114 9329
-3997 -7377
-7318 8405
-9222 -1494
-5672 -8698
-4618 -1793
6512 1703
-195 6269
2171 5990
7111 -1364
-5284 6500
-8789 2784
5695 -11
-8583 9455
-7935 3056
-2657 -6714
7187 -5485
5947 8462
1643 -4959
5253 2798
-8636 9287
4917 7919
7914 -543
-5833 2488
