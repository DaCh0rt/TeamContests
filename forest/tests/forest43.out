78788.10 401188585.30
